src/compas/datastructures/network/network.py

```diff
@@ -209,6 +209,30 @@ def from_nodes_and_edges(cls, nodes, edges):
 
         return network
 
+    @classmethod
+    def from_pointcloud(cls, cloud, degree=3):
+        """Construct a network from random connections between the points of a pointcloud.
```

Comment:
Interesting method. I think the name does not clearly imply this is a random network, I'm not sure if it's needed to either.  Perhaps it wouldn't be a bad idea to add a way to create a network from point cloud, but not connect it randomly by default. However, I do see that it's also too trivial to justify.